website/docs/policies/expression.mdx

```diff
@@ -54,6 +54,11 @@ import Objects from "../expressions/_objects.md";
     -   `request.context`: A dictionary with dynamic data. This depends on the origin of the execution.
 
 -   `geoip`: GeoIP object, see [GeoIP](https://geoip2.readthedocs.io/en/latest/#geoip2.models.City)
+
+    ```python
+    return context["geoip"].country.iso_code == "US"
+    ```
+
 -   `ak_is_sso_flow`: Boolean which is true if request was initiated by authenticating through an external provider.
 -   `ak_client_ip`: Client's IP Address or 255.255.255.255 if no IP Address could be extracted. Can be [compared](#comparing-ip-addresses), for example
```

Comment:
Is there supposed to be more at the end of this line? Or do we just need a period there at end?

And can we rewrite to "If no IP Address could be extracted, this value defaults to either the client's IP Address or 255.255.255.255. You can use the [compare functions](#comparing-ip-addresses) to learn more about the IP address."